quickstart/docker/simplified-docker-compose.yml

```diff
@@ -37,6 +39,8 @@ services:
     environment:
       - ZAC_SERVER_CERT_CHAIN=/openziti/pki/ziti-controller-intermediate/certs/ziti-controller-server.cert
       - ZAC_SERVER_KEY=/openziti/pki/ziti-controller-intermediate/keys/ziti-controller-server.key -it
```

Comment:
this ` -it` appears to be a stray copy/paste error? can you clean that up if yes?